RestSharp/Http.cs

```diff
@@ -430,18 +426,24 @@ private void ExtractResponseData(HttpResponse response, HttpWebResponse webRespo
                     });
                 }
 
+                var webResponseStream = webResponse.GetResponseStream();
+                ProcessResponseStream(webResponseStream, response);
+
                 webResponse.Close();
             }
         }
 
         private void ProcessResponseStream(Stream webResponseStream, HttpResponse response)
         {
-            if (ResponseWriter == null)
-                response.RawBytes = webResponseStream.ReadAsBytes();
+            if (AdvancedResponseWriter != null)
```

Comment:
What is the purpose of this change? For this method, the previous code did exactly the same, didn't it?

Reply:
It did not to the same thing. When ResponseWriter is null and AdvancedResponseWriter is non-null, stream will be consumed by ReadAsBytes() call.